components/library/libfreehand/Makefile

```diff
@@ -23,17 +23,17 @@
 BUILD_BITS=32_and_64
 include ../../../make-rules/shared-macros.mk
```

Comment:
/usr/bin/pkgdepend generate -m -d /export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled -d /export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386 -d /export/home/andreas/oi-userland/components/library/libfreehand/build -d /export/home/andreas/oi-userland/components/library/libfreehand -d libfreehand-0.1.2 /export/home/andreas/oi-userland/components/library/libfreehand/build/manifest-i386-libfreehand.mangled >/export/home/andreas/oi-userland/components/library/libfreehand/build/manifest-i386-libfreehand.depend
Couldn't find 'usr/share/doc/libfreehand/html/annotated.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/classPerfect__Hash-members.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/classPerfect__Hash.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/classes.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/functions.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/functions_func.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/functions_vars.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/globals_defs.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/structfhtoken-members.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
Couldn't find 'usr/share/doc/libfreehand/html/structfhtoken.html' in any of the specified search directories:
	/export/home/andreas/oi-userland/components/library/libfreehand
	/export/home/andreas/oi-userland/components/library/libfreehand/build
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386
	/export/home/andreas/oi-userland/components/library/libfreehand/build/prototype/i386/mangled
	/export/home/andreas/oi-userland/components/library/libfreehand/libfreehand-0.1.2
gmake: *** [/export/home/andreas/oi-userland/make-rules/ips.mk:498: /export/home/andreas/oi-userland/components/library/libfreehand/build/manifest-i386-libfreehand.depend] Error 1